### PR TITLE
Implemented StudySession

### PR DESCRIPTION
**Pull request by cdw18**
Implemented StudySession and took care of the multiple issues from phase 0 (issue #1 )

### CODE REVIEW

File: Uni_life_tracker/app/src/main/java/Entities/StudySession.java [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+import java.time.LocalDateTime;
 
-public class StudySession {
+/**
+ * Represents a study session of a course, with participants, location and a note.
+ */
+public class StudySession extends Event {
     private String course;
     private String location;
     private String notes;
```
Comment (sarihammad) [inline Uni_life_tracker/app/src/main/java/Entities/StudySession.java]: I think we decided that notes was no longer an instance variable for any of the event types.

**Review by sarihammad — CHANGES_REQUESTED** [review]
I think we decided to remove notes as an instance variable from Events. Other than that, all looks good!

File: Uni_life_tracker/app/src/main/java/Entities/StudySession.java (hunk 1 above)
Comment (cdw18) [reply Uni_life_tracker/app/src/main/java/Entities/StudySession.java]: I actually thought about removing it. I'll get working on that.

**Review by OKok-3 — APPROVED** [review]

**Review by sarihammad — APPROVED** [review]
👍

**Pull request merged**